dotcom-rendering/src/web/layouts/InteractiveLayout.tsx

```diff
@@ -531,6 +535,42 @@ export const InteractiveLayout = ({ CAPIArticle, NAV, format }: Props) => {
 										isDev={!!CAPIArticle.config.isDev}
 									/>
 
+									<Island clientOnly={true}>
```

Comment:
I'm not completely sure of the flow here but I _think_ this is roughly where you want to have `ContainerLayout`. I can't be sure though as it depends on if the slot will always show or only sometimes